setup.cfg

```diff
@@ -1,3 +1,16 @@
 [bdist_wheel]
 # code is written to work on both Python 2 and Python 3
-universal=1
+universal = 1
+
+[pep8]
+max-line-length = 119
```

Comment:
I would prefer if we would [follow PEP8](https://www.python.org/dev/peps/pep-0008/#maximum-line-length) and limit the maximum line length to 100.

How much work would it require?